tensorflow/compiler/mlir/tosa/tests/tf-to-tosa-pipeline.mlir

```diff
@@ -105,9 +105,16 @@ func @test_relu6(%arg0: tensor<13x21x3xf32>) -> tensor<13x21x3xf32> {
 // -----
 
```

Comment:
I didn't double check, but I think that some of the op conversions added are missing tests (gather?). Please audit test coverage for both ops and critical branches in the patterns.

Reply:
Thanks. Will fix in next round.

Reply:
Still missing tests for gather support. Because we are adding functionality for a gather lowering we will need a test to validate the lowering is working correctly.